hw/ip/clkmgr/data/clkmgr.sv.tpl

```diff
@@ -0,0 +1,179 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+//
+// The overall clock manager
+
+`include "prim_assert.sv"
+
+<%
+clks_attr = cfg['clocks']
+srcs = clks_attr['srcs']
+grps = clks_attr['groups']
+
+## feedthrough clocks
+ft_clks   = {clk:src for grp in grps for (clk,src) in grp['clocks'].items()
```

Comment:
is this too hard to read?

Reply:
yup. rather, `{clk:src for (clk,src) in grp['clocks'].items() for grp in grps}` and filter the grp['name'] powerup prior to this line.

Reply:
And, if this filtering is needed, I think it is also good to create these fields in the data structure in `clkmgr_gen`.

Reply:
But fine for now

Reply:
btw, in terms of the ordering of `list` construction, is the outter loop supposed to be first or second? Because i actually found examples where the outer loop, ie `for grp in grps` is first. 

Let me see if we have style guides about this...

Reply:
@eunchan, when you say filter prior to this line, do you mean i should filter into a new list first? That does make sense...